package.json

```diff
@@ -42,5 +42,17 @@
   },
   "devDependencies": {
     "sass": "^1.42.1"
-  }
+  },
+  "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
```

Comment:
dude, could you please remove these added lines.

Reply:
oh yes I am sorry